Dockerfile

```
@@ -1 +1,2 @@
-FROM botpress/server:latest
\ No newline at end of file
+FROM botpress/server:latest
+CMD ["./bp", "&" ,"./duckling"]
```

Shouldn't it be 

```suggestion
CMD ["./bp", "&&" ,"./duckling"]
```

or maybe 

```suggestion
CMD ["./duckling", "&" ,"./bp"]
```

?

---

No Heroku take the first process and use that port to expose the web application